pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -379,7 +379,7 @@ cdef class {{name}}HashTable(HashTable):
                 self.table.vals[k] = i
 
     @cython.boundscheck(False)
-    def lookup(self, {{dtype}}_t[:] values):
+    def lookup(self, const {{dtype}}_t[:] values):
```

Comment:
we have multiple lookup routines for different drapes - is there a reason to not change them all?

why just lookup? we have many routines which couldnin theory benefit from this

Reply:
I can make a pass over `hashtable_class_helper.pxi.in` and add `const` specifiers where applicable.